client/components/Stars.jsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+// creates stars image that reflects reviews stars count
+function Stars({ starCount }) {
+  if (starCount === 1) {
+    return (
+      <div className="stars_wrapper">
+        <i className="stars_1" />
+      </div>
+    );
+  }
+  if (starCount === 2) {
+    return (
+      <div className="stars_wrapper">
+        <i className="stars_2" />
+      </div>
+    );
+  }
+  if (starCount === 3) {
+    return (
+      <div className="stars_wrapper">
+        <i className="stars_3" />
+      </div>
+    );
+  }
+  if (starCount === 4) {
+    return (
+      <div className="stars_wrapper">
+        <i className="stars_4" />
+      </div>
+    );
+  }
+  if (starCount === 5) {
+    return (
+      <div className="stars_wrapper">
+        <i className="stars_5" />
+      </div>
+    );
+  }
+  if (starCount === 0) {
+    return (
+      <div className="stars_wrapper">
+        <i className="stars_0" />
+      </div>
+    );
+  }
+}
```

Comment:
Another option for this might be to use a switch statement in your RowB.jsx file that selects the correct className and then renders the div in that file. It might also be possible to combine the starCount variable with a string to form the new className in the RowB.jsx file

`<div className="stars_wrapper">
        <i className=`stars_${stars}` />
  </div>`